src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -544,4 +581,159 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
         // add all accounts needing updating to our list
         dmlWrapper.objectsToUpdate.addAll((List<SObject>)listAccToUpdate);
     }
+
+    /*******************************************************************************************************
+    * @description format the account name based on naming settings
+    * @param cons the Contacts need to format Account Name
+    * @param setting the Account naming setting
+    * @return string. the formatted account name
+    */
+
+    private static String formatAccountName(List<Contact> cons, String setting) {
+        //get all the fields of Contact
+        Map<String, Schema.SObjectField> mapContactFields = Schema.getGlobalDescribe().get('Contact').getDescribe().fields.getMap();
```

Comment:
I think this method will be call by a For loop in multiple places, maybe we should consider to use UTIL_Describe to get the fields instead of call the global describe.

Reply:
I find there is a method in UTIL_Describe class called isValidField. I think we can use it directly without calling schema method here. Although it will still call the global describe a couple of times, I am not too concerned about it since they are super lightweight tokens. What do you think?

Reply:
Good finding in the UTIL_Describe class!  It will be great if we can use it. I think the describe result in the UTIL_Describe is cached, so we should be fine.

Reply:
I just love that hooray icon so much! Ok I will make the changes now. Thanks @weizsforg!